targets/ath79-generic

```diff
@@ -76,3 +76,8 @@ device('ocedo-raccoon', 'ocedo_raccoon', {
 device('tp-link-archer-c6-v2', 'tplink_archer-c6-v2', {
 	packages = ATH10K_PACKAGES_QCA9888,
 })
+
+device('tp-link-archer-d50-v1', 'tplink_archer-d50-v1', {
+        packages = ATH10K_PACKAGES_QCA9880,
+        factory = false,
```

Comment:
Device has only 64M of memory, which means it has to be flagged as BROKEN.

Reply:
Why its needs to be marked as broken? The docs dose not show requirements like that?

Reply:
It is indeed not documented. This is due to the fact the devices OOM very often when using the 5GHz radio.

Upstream removed the patch which reduced memory footprint in https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=1e27befe63ff4c69f110c7310316f4af75ee63e9 . So the devices will be barely usable in the next release.

We can't fallback on the -ct-smallbuffers variant, as this does not support mesh.

Reply:
But it will be possible to use 2.4 GHz or 5GHz only? 
So why not switch from AND to OR in the wireless settings?

Reply:
hm, I see ... but the idea from @jenell95 is good. Maybe we should offer thous devices with default 5GHz enabled and 2,4GHz disabled. But anyway this is getting off topic. I will add a broken (or legacy) flag at first. A general solution how to handle thous kind of problems like @jenell95 advised can be discuss in an separated issue.